dist/templates/k8s.ovn.org_adminpolicybasedexternalroutes.yaml.j2

```diff
@@ -112,11 +112,11 @@ spec:
                       field is optional.
                     items:
                       description: DynamicHop defines the configuration for a dynamic
-                        external gateway interface. These interfaces are wrapped around
-                        a pod object that resides inside the cluster. The field NetworkAttachmentName
-                        captures the name of the multus network name to use when retrieving
-                        the gateway IP to use. The PodSelector and the NamespaceSelector
-                        are mandatory fields.
+                        external gateway interface.ss These interfaces are wrapped
```

Comment:
typo?: `ss`

Reply:
Updated. I rebuilt the schema and made sure the original copy of the go file I have matches the one from this repo, so next time I have to rebuild it I don't accidentally delete/add artifacts or annotations.